ktor-server/ktor-server-core/api/ktor-server-core.api

```diff
@@ -763,7 +763,7 @@ public final class io/ktor/server/response/DefaultResponsePushBuilder : io/ktor/
 }
 
 public final class io/ktor/server/response/ResponseCookies {
-	public fun <init> (Lio/ktor/server/response/ApplicationResponse;Z)V
+	public fun <init> (Lio/ktor/server/response/ApplicationResponse;)V
```

Comment:
Can we put this change in the patch release?

Reply:
I can leave in the unused boolean argument here to maintain compatibility.